Objects/memoryobject.c

```diff
@@ -2297,6 +2299,8 @@ memoryview_tobytes_impl(PyMemoryViewObject *self, const char *order)
 }
 
 /*[clinic input]
+@text_signature "($self, *, bytes_per_sep=1, /)"
+@text_signature "($self, sep, bytes_per_sep=1, /)"
```

Comment:
```suggestion
@text_signature "($self, *, bytes_per_sep=1)"
@text_signature "($self, sep, bytes_per_sep=1)"
```

Reply:
Done (`$self` should be positional-only).

Reply:
Yes, it should always be positional-only. It is enforced in normal Argument Clinic input blocks (but not for overridden text signatures).